integ/README.md

```diff
@@ -3,22 +3,43 @@
 To run all test suites:
 
 1. Build and install dependencies by running build.sh from the top-level RFDK directory
+1. Configure AWS credentials (tests will use the default AWS profile, so either set up a default profile in `~/.aws/`
+   credentials or use temporary credentials).
+1. Set the environment variable CDK_DEFAULT_REGION to the region the test should be deployed in
+1. Modify the `test-config.sh` configuration file. Currently the following options can be configured:
```

Comment:
I'm surprised that we don't have an env var corresponding to: https://docs.aws.amazon.com/rfdk/api/latest/docs/aws-rfdk.deadline.ThinkboxDockerImages.html#userawsthinkboxeulaacceptance

Reply:
These are instructions for running the tests using the `test-config.sh` configuration file. All the variables in that file (including `USER_ACCEPTS_SSPL_FOR_RFDK_TESTS`) can be set using environment variables, but in order to avoid the config file from overriding the environment variables, an extra environment variable can be set:

```sh
export SKIP_TEST_CONFIG=true
```

Reply:
Right. Sorry, I might not have been clear. We have an env var for the MongoDB SSPL, but I gather we don't have one for the Deadline license? Is the integ test just hard-coded to always accept that license?